### ISSUE TITLE
Error in ytdl

### ISSUE THREAD

**Issue by robotrono** [mps-youtube/pafy#116]
Ubuntu 14.04 x64 / VPS 
```
# ytdl cyMHZVT91Dw
Traceback (most recent call last):    <
  File "/usr/local/bin/ytdl", line 194, in <module>
    main()
  File "/usr/local/bin/ytdl", line 185, in main
    printstreams(streams)
  File "/usr/local/bin/ytdl", line 103, in printstreams
    megs = "%3.f" % (s.get_filesize() / 1024 ** 2) + " MB"
TypeError: unsupported operand type(s) for /: 'NoneType' and 'int'
```

**Comment by ids1024** [comment]
Looks like the same issue as #114, which I am looking into right now.

**Comment by ids1024** [comment]
Fixed in commit ac4df11990592e67a04afc81914f5ad364100030.
